recipe/build.sh

```diff
@@ -38,6 +38,7 @@ export TF_SYSTEM_LIBS="
   astor_archive
   astunparse_archive
   boringssl
+  com_google_absl
   com_github_googlecloudplatform_google_cloud_cpp
   com_github_grpc_grpc
   com_google_protobuf
```

Comment:
This list was alphabetical so far, please sort `com_google_absl` at the appropriate place.

Also, seeing the comment just above `TF_SYSTEM_LIBS`, we now have C++17 re2 in conda-forge.

Reply:
Added both things as tasks in the top comment. I will check that once a clean build with `com_google_absl` has passed.